follow-ups/arrays-objects/index.html

```diff
@@ -186,13 +189,16 @@ <h3 style="font-size: 3rem">Rozmiar tablicy</h3>
           <h3>Pętle!</h3>
           <p class="fragment">Przypomnijmy sobie pętle z warsztatu.</p>
           <pre class="fragment"><code class="hljs javascript">for (let i=0; i<10; i++) {
-  // wykonaj ten kod tak długo jak warunek trwania pętli jest prawdziwy
+  // wykonaj ten kod tak długo
+  // jak długo warunek trwania pętli jest prawdziwy
 }</code></pre>
         </section>
 
         <section>
           <h3>Spróbujcie wpisać w konsolę następujący kod:</h3>
-          <pre><code class="hljs javascript">for (var i=0; i<5; i++) { console.log(i) }</code></pre>
+          <pre><code class="hljs javascript">for (var i=0; i<5; i++) {
+  console.log(i);
```

Comment:
Na slajdzie jest duzo miejsca.
Pokazujmy przykłady, ktore "promują" dobre formatowanie kodu

Reply:
Teraz, kiedy dev toolsy są mądre i same potrafią odróżniać łamanie linii od zatwierdzenia, masz całkowitą rację.

Reply:
Tak, przeczytalem wlasnie zdanie w naglowku.
Wroce do poprzedniego.

Arturze, przedkladam komunikacje wprost ponad błyskotliwe sarkazmy.
Szczegolnie w przypadku projektow, na ktore poswiecam swoj prywatny czas.
Pozdrawiam

Reply:
Maciek, nie wiem gdzie się doszukujesz sakrazmu, ale właśnie teraz dev toolsy są na tyle mądre, że odróżniają łamanie linii od zatwierdzenia. Serio. Jak mawiają, nie mierz innych swoją miarą. To, że Ty często posługujesz się sarkazmem, nie znaczy, że robią tak inni ;-)

Reply:
A w sumie, to w chrome tak potrafią, nie wiem jak safari czy ffox. Sprawdzę.

Reply:
Odtwarzam stara wersje tak czy siak

Reply:
U mnie Safari 11 i FF 55.0.3 umiom.
Jestem w kontekscie konsoli i skonfudowany w kontekscie formatowania tego kodu w prezentacji.